File: fingertip/src/main/java/com/praszapps/easyfingerprint/view/FingertipActivity.java
package com.praszapps.easyfingerprint.view;

import android.os.Bundle;
import android.support.annotation.Nullable;

import com.praszapps.easyfingerprint.MVP.FingertipMVPContract;
import com.praszapps.easyfingerprint.presenter.FingertipActivityPresenter;

import javax.inject.Inject;

import dagger.android.support.DaggerAppCompatActivity;

public class FingertipActivity extends DaggerAppCompatActivity implements FingertipMVPContract.View {

    @Inject
    FingertipActivityPresenter mPresenter;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public void setUpFingerprintViews() {

    }

    @Override
    public void handleNoFingerprintAuthPossible() {

    }

    @Override
    public void onFingerPrintAuthenticationSuccess() {

    }

    @Override
    public void onFingerprintAuthenticationFailed(String text) {

    }
}

File: fingertip/src/main/java/com/praszapps/fingertip/view/FingertipActivity.kt
package com.praszapps.fingertip.view

import com.praszapps.fingertip.MVP.FingertipMVPContract
import dagger.android.support.DaggerAppCompatActivity
import javax.inject.Inject
import javax.inject.Named

class FingertipActivity : DaggerAppCompatActivity(), FingertipMVPContract.View {

    @Inject
    @Named("activityPresenter")
    lateinit var mPresenter: FingertipMVPContract.Presenter

    override fun setUpFingerprintViews() {

    }

    override fun handleNoFingerprintAuthPossible() {

    }

    override fun onFingerPrintAuthenticationSuccess() {

    }

    override fun onFingerprintAuthenticationFailed(text: String) {

    }
}
